rules/docker-sonic-sdk-buildenv.mk

```diff
@@ -28,5 +28,5 @@ $(DOCKER_SONIC_SDK_BUILDENV)_LOAD_DOCKERS += $(DOCKER_SONIC_SDK)
 SONIC_DOCKER_IMAGES += $(DOCKER_SONIC_SDK_BUILDENV)
 SONIC_DOCKER_DBG_IMAGES += $(DOCKER_SONIC_SDK_BUILDENV_DBG)
 
-SONIC_BUSTER_DOCKERS += $(DOCKER_SONIC_SDK_BUILDENV)
-SONIC_BUSTER_DBG_DOCKERS += $(DOCKER_SONIC_SDK_BUILDENV_DBG)
+SONIC_DOCKER_IMAGES += $(DOCKER_SONIC_SDK_BUILDENV)
```

Comment:
Should these be `SONIC_BULLSEYE_DOCKERS` and `SONIC_BULLSEYE_DBG_DOCKERS` instead?

Reply:
I think no, This has to be under SONIC_DOCKER_IMAGES https://github.com/sonic-net/sonic-buildimage/blob/master/slave.mk#L961 && https://github.com/sonic-net/sonic-buildimage/blob/master/slave.mk#L949 

Although, i see some targets were added to SONIC_BULLSEYE_DOCKERS but the variable is not used anywhere. Do you want me to also include this variable for these targets?

Reply:
`SONIC_DOCKER_IMAGES` is already specified two lines above here (lines 28 and 29).

You're correct in that `SONIC_BULLSEYE_DOCKERS` is currently not used anywhere, and that some of the container upgrade PRs added those containers to this variable. I'm fine either way if it's added here or not.

Reply:
Oh, sry. I've overlooked it. Updated